content/docs/glossary/fee-bumps.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Fee-Bump Transactions
+order:
+---
+A fee-bump transaction enables any account to pay the fee for an existing [transaction](./transactions.mdx) without the need to re-sign the existing transaction or manage sequence numbers.  They're useful if you need to increase the fee on an pre-signed transaction, or if you want to build a service that covers user fees.  Unlike a regular transaction, which contains 1-100 [operations](./operations.mdx), a fee-bump transaction contains a single [transaction enevelope](./transactions.mdx/#transaction-envelopes).     
```

Comment:
Nitpick: "an pre-signed" -> "a pre-signed"

Reply:
Fixed!  Thank you.